week_2/ec2.tf

```diff
@@ -0,0 +1,22 @@
+variable "instance_name" {
```

Comment:
Also, please consider creating `output.tf` and output EC2 public ip

Reply:
Added
![26B69440-0C03-440E-8E93-4D2DE86E9133](https://user-images.githubusercontent.com/28685157/165487030-987c9ae6-c174-467a-b707-e7d868ae5804.jpeg)